docs/dev/metrics.md

```
@@ -4,19 +4,28 @@ The Cluster Machine Approver reports the following metrics:
 
 ## Metrics about pending certificate signing requests (CSRs)
 
-These metrics show how many CSRs are currently pending as well as the
-maximum number allowed to be pending. These can be useful to help diagnose
-the flow of new Nodes being added to the cluster.
+These metrics show how many recently pending node CSRs are currently counted
+as well as the threshold used by MachineApproverMaxPendingCSRsReached.
+They help diagnose Node bootstrap and CSR approval behavior during scale-up.
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)